yaksh/documentation/moderator_docs/creating_course.rst

```diff
@@ -37,21 +40,81 @@ Features in Courses
 
      The following features are available for courses
 
-        * Course Name
+        * **Course Name**
             Clicking on course name link will display all the enrolled, rejected and requested students list. Moderator can accept or reject the student.
-        * Quiz Name
-            Clicking on the quiz name will let you edit the quiz.
-        * Question Paper
-            Click on the **Add** link to create a Question Paper for associated Quiz.
-            If a question paper is already created, click on the Question Paper link to edit question paper.
-        * Add Teacher
+        * **Module Name**
+            Click to edit a module added to the course
+        * **Lesson or Quiz Name**
+            Click to edit a Lesson or Quiz added to the course
+
+            In edit quiz you can also attempt the quiz in two modes - 
+                * **God Mode** - In God mode you can attempt quiz without any time or eligibilty constraints.
+                * **User Mode** - In user mode you can attempt quiz the way normal users will attempt i.e.
+                    
+                    * Quiz will have the same duration as that of the original quiz.
+                    * Quiz won't start if the course is inactive or the quiz time has expired.
+                    * You will be notified about quiz prerequisites.(You can still attempt the quiz though)
+        * **Add Quizzes/Lessons for <module-name>**
+            Click to add/change/delete lessons or quizzes.
+        * **My Courses**
+            Clicking on this will show all the courses created by you.
+        * **Allotted courses**
+            Clicking on this will show all the courses allotted to you.
+        * **Add New Course**
+            Clicking on this will open course form to create new course.
+        * **Add/View Quizzes**
+            Click to view all the created quizzes and add new quizzes.
+        * **Add/View Lessons**
+            Click to view all the created lessons and add new lessons.
+        * **Add/View Modules**
+            Click to view all the created modules and add new modules.
+        * **Design Course**
+            Click to add/delete/change modules.
+        * **Add Teacher**
             Clicking on Add teacher can let you add teachers for the course. The teachers can edit and modify only the specific course that are allotted to them.
-        * Teachers added to the course
+        * **Clone Course**
+            This will create a clone of the course for the moderator.
```

Comment:
Clicking on `Clone Course` will create a clone of the entire course for the moderator.